mirrord-layer/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![feature(naked_functions)]
 #![feature(result_flattening)]
 #![feature(io_error_uncategorized)]
+#![feature(let_chains)]
```

Comment:
Do we need to enable this feature? I'm on nightly `rustc 1.65.0-nightly (bc4b39c27 2022-08-29)` and it doesn't warn my anywhere.

Reply:
mm when I updated it does that.. let's leave it for now I guess.